### DISCUSSION TITLE
Renaming the master branch to main

### DISCUSSION THREAD

**Discussion by C4ptainCrunch** [ics-py/ics-py#279] [Show and tell]
I'm changing the master branch to main. Nothing really changes, you just have to update your local copy with these commands:
```bash
git branch -m master main
git fetch origin
git branch -u origin/main main
```

**Comment by tomschr** [comment]
Just to let you know: you can do that in GitHub.

Go to your "Settings > Branches" entry. Under "Default branch" there is a little "edit" icon on the right side. If you click there, GitHub opens a popup window where you can rename the branch. Additionally, it takes care of open pull requests. They are updated.

Here is a short screenshot:

![gh-rename-branch](https://user-images.githubusercontent.com/1312925/104844066-332c9880-58ce-11eb-8fa1-fb802e38530f.png)

**Reply by C4ptainCrunch** [reply]
That's what i did, but thank you nevertheless ! :+1:

**Comment by tomschr** [comment]
Yeah, I realized that I came too late to the party.  :laughing:  You did that already. :+1: 

Maybe you want to mark this discussion as solved? :wink:
